### DISCUSSION TITLE
How to export vector graphics from 3D visualizer?

### DISCUSSION THREAD

**Discussion by fanlegeshao** [mzmine/mzmine#1167] [Q&A]
MZmine3 is fantastic! I tried it for some time and really like it.
But I was wondering if I could export vector graphics (or other type) from 3D visualizer as the homepage described.

**Comment by robinschmid** [comment]
As far as I know, this is not possible for the 3D chart. @tomas-pluskal do you know?
You can export all 2D charts as vector or pixel graphics. Right-click and choose "Graphics export dialog".

**Reply by tomas-pluskal** [reply]
> As far as I know, this is not possible for the 3D chart. @tomas-pluskal do you know? 

No, there is no export from the 3D plot, you can only take a screenshot.

**Comment by fanlegeshao** [comment]
Wowwww, thanks for your remind. That will be awesome too. 😉



---Original---
From: "Robin ***@***.***&gt;
Date: Tue, Feb 28, 2023 00:22 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [mzmine/mzmine3] How to export vector graphics from 3Dvisualizer? (Discussion #1167)




 
As far as I know, this is not possible for the 3D chart. @tomas-pluskal do you know?
 You can export all 2D charts as vector or pixel graphics. Right-click and choose "Graphics export dialog".
